CollapseLauncher/Classes/Properties/ArgumentParser.cs

```diff
@@ -118,41 +120,47 @@ public static void ParseArguments(params string[] args)
             m_appMode = AppMode.Launcher;
         }
 
-        public static void ParseHi3CacheUpdaterArguments(params string[] args)
+        private static void ParseHi3CacheUpdaterArguments(params string[] _)
         {
             Command hi3CacheUpdate = new Command("hi3cacheupdate", "Update the app or change the Release Channel of the app");
-            _rootCommand.AddCommand(hi3CacheUpdate);
+            _rootCommand.Add(hi3CacheUpdate);
             AddHi3CacheUpdaterOptions(hi3CacheUpdate);
         }
 
-        public static void ParseUpdaterArguments(params string[] args)
+        private static void ParseUpdaterArguments(params string[] _)
         {
             Command updater = new Command("update", "Update the app or change the Release Channel of the app");
-            _rootCommand.AddCommand(updater);
+            _rootCommand.Add(updater);
             AddUpdaterOptions(updater);
         }
 
-        public static void ParseElevateUpdaterArguments(params string[] args)
+        private static void ParseElevateUpdaterArguments(params string[] _)
         {
             Command elevateUpdater = new Command("elevateupdate", "Elevate updater to run as administrator");
-            _rootCommand.AddCommand(elevateUpdater);
+            _rootCommand.Add(elevateUpdater);
             AddUpdaterOptions(elevateUpdater);
         }
 
-        public static void AddHi3CacheUpdaterOptions(Command command)
+        private static void AddHi3CacheUpdaterOptions(Command _) { }
+
+        private static void AddUpdaterOptions(Command command)
         {
-            command.SetHandler(() =>
+            var oInput = new Option<string>("--output-path")
             {
-            });
-        }
+                Description = "Output path for the output file",
+                Required    = true
+            };
 
-        public static void AddUpdaterOptions(Command command)
-        {
-            Option<string>            oInput   = new(["--input", "-i"], "App path") { IsRequired = true };
-            Option<AppReleaseChannel> oChannel = new Option<AppReleaseChannel>(["--channel", "-c"], "App release channel") { IsRequired = true }.FromAmong();
-            command.AddOption(oInput);
-            command.AddOption(oChannel);
-            command.Handler = CommandHandler.Create((string input, AppReleaseChannel channel) =>
+            var oChannel = new Option<AppReleaseChannel>("--channel", "-c")
```

Comment:
Make this in `Aliases` style?

Reply:
Done https://github.com/CollapseLauncher/Collapse/commit/3e15b13e9adebcfc752f702d0cafd86b0d3469f4